packages/semaphore-identity/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zk-kit/semaphore-identity-artifacts",
-  "version": "1.0.0-beta",
+  "version": "1.0.0-beta.2",
```

Comment:
to match https://github.com/semaphore-protocol/extensions/blob/0f81734ef8ab1c4f0770536bb8519ce0ec683cfd/packages/identity-proof/package.json#L3